.github/workflows/install.yml

```
@@ -41,14 +50,25 @@ jobs:
           sudo incus admin init --auto
           sudo incus launch ${{ matrix.image }} target
           sudo incus config device add target host disk source=$PWD path=/host
-          # Wait for network services stabilizing
-          sleep 1 # Wait for systemd ready
-          sudo incus exec target -- sudo systemctl is-system-running --wait
+          # Ideally, we would use systemctl is-system-running --wait to ensure all services are fully operational.
+          # However, this option doesn't work in AlmaLinux 8 and results in an error.
+          # As a workaround, we introduced a 10-second sleep delay to allow network services time to stabilize,
+          # preventing DNS resolution errors when attempting to dnf install command.
+          # ref: https://discuss.linuxcontainers.org/t/network-issue-with-almalinux-8-9-on-github-actions-using-incus/20046
+          sleep 10
       - name: Install Mroonga
         run: |
+          case ${{ matrix.image }} in
+            images:ubuntu/*)
+              package_type=apt
+              ;;
+            *)
+              package_type=yum
+              ;;
+          esac
           sudo incus exec target \
             -- \
-            /host/packages/apt/install_test.sh \
+            /host/packages/${package_type}/install_test.sh \
```

```suggestion
            "/host/packages/${package_type}/install_test.sh" \
```
変数があるやつは `"` で囲むのが無難っぽいです。

---

おぉ。そうなのですね。。。

`case ${{ matrix.image }} in`とかも`case "${{ matrix.image }}" in`みたいにしたほうが良いのですね。

---

変数があるやつを`"`で囲みました。